simp_le.py

```diff
@@ -850,6 +850,10 @@ def create_parser():
         '--reuse_key', action='store_true', default=False,
         help='Reuse private key if it was previously persisted.',
     )
+    io_group.add_argument(
+        '--renew_different_san', action='store_true', default=False,
+        help='Don\'t abort if the SANs changed'
```

Comment:
I think `--allow_domain_changes` is the most suitable of the proposed ones. @kuba?